src/panels/lovelace/common/process-config-entities.ts

```diff
@@ -1,7 +1,8 @@
 // Parse array of entity objects from config
 import isValidEntityId from "../../../common/entity/valid_entity_id";
+import { EntityConfig } from "../entity-rows/types";
 
-export default function processConfigEntities(entities) {
+export const processConfigEntities = (entities: EntityConfig[]) => {
```

Comment:
This is not correct. The output is `EntityConfig[]`, the input can also be a string (as tested for on line 19) so should be `Array<EntityConfig | string>`. Also please mark the return type

Reply:
I think that it should be something like this:

```suggestion
export const processConfigEntities = <T extends EntityConfig>(entities: Array<EntityConfig | string>[]): T[] => {
```